cmd/queue/main.go

```diff
@@ -296,6 +296,11 @@ func buildServer(ctx context.Context, env config, healthState *health.State, pro
 			}
 		}()
 		composedHandler = queue.ConcurrencyStateHandler(logger, composedHandler, ce.Pause, ce.Resume)
+		// start paused
+		if err := ce.Pause(); err != nil {
+			logger.Errorf("Error handling initial start request: %v", err)
```

Comment:
```suggestion
			logger.Errorf("Error handling initial pause request: %v", err)
```

Reply:
future suggestion: we might want to defer the initial pause until we see the container go ready (so we pause after user container startup is complete)

Reply:
Don't we **have** to do that? Will we ever report ready without delaying it?

Reply:
my mental model is we'll see it, but all the startup will happen during the health check (while resumed), and we'll definitely risk timeouts

Reply:
Anecdotally, I haven't seen issues when running this locally. That said, if we want to put the defer in this PR (rather than a future one), I'm happy to do so.